torchdynamo/symbolic_convert.py

```diff
@@ -645,6 +645,21 @@ def LOAD_ATTR(self, inst):
                     self.push(UserMethodVariable(method, obj, **options))
                 else:
                     unimplemented("nn.Module callable")
+            elif isinstance(subobj, list):
+                # If the list contains exclusively nn.Modules, transform
+                # each module into a torchdynamo.NNModuleVariable
+                l = []
+                for i, item in enumerate(subobj):
+                    if isinstance(item, torch.nn.Module):
+                        key = f"{name}_{i}"
+                        self.nn_modules[key] = item
+                        l.append(NNModuleVariable(
+                                     module_key=key,
+                                     state=TracingSupported.YES,
+                                     ))
+                if len(l) == len(subobj):
```

Comment:
I think if this check fails we should call `unimplemented("....")`  (that means there is some unsupported things in the list)